File: 0001-two-sum/0001-two-sum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        let counter = nums.count;
        var result : [Int] = [];
        let number = 9;
        var lister : [String] = [];
        for i in 0..<counter
        {
            for y in (i+1)..<counter
            {
                if nums[i]+nums[y] == target {
                    result =  [i, y]
                    lister.append("hello")
                    print(lister)
                }
            }
        }
        return result
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        
        for n in range(len(nums)):
            for s in range(n + 1, len(nums)):
                if nums[n] + nums[s] == target:
                    return [n, s]
            
        return [0, 0]